src/braket/experimental/autoqasm/pulse/pulse.py

```diff
@@ -127,19 +128,21 @@ def capture_v0(frame: Frame) -> None:
 
 def delay(
     qubits_or_frames: Union[Frame, List[Frame], QubitIdentifierType, List[QubitIdentifierType]],
-    duration: float,
+    duration: Union[float, oqpy.FloatVar],
```

Comment:
In an `@aq.gate_calibration` function like this:
```
    @aq.gate_calibration(implements=rx)
    def cal_2(angle: float):
        pulse.delay("$1", angle)
```

AutoQASM transpiles the `angle` param as an `oqpy.FloatVar` here, so that's why `delay` accepts `FloatVar` (and then converts it to a `FreeParameter`).